avocado/core/job.py

```diff
@@ -323,22 +323,14 @@ def _log_cmdline():
 
     @staticmethod
     def _log_avocado_version():
-        LOG_JOB.info('Avocado version: %s', version.VERSION)
-        if os.path.exists('.git') and os.path.exists('avocado.spec'):
+        version_log = version.VERSION
+        if os.path.exists('.git') and os.path.exists('python-avocado.spec'):
```

Comment:
You can add some more flexibility by using `git rev-parse --show-toplevel` instead of `os.path.exists('.git')`, so it will work from any sub-directory inside the git tree.